Marlin/src/config/examples/Creality/Ender-4/Configuration.h

```diff
@@ -661,22 +661,22 @@
  * Override with M92
  *                                      X, Y, Z, E0 [, E1[, E2[, E3[, E4[, E5]]]]]
  */
-#define DEFAULT_AXIS_STEPS_PER_UNIT    { 80, 80, 400, 110 }
+#define DEFAULT_AXIS_STEPS_PER_UNIT    { 80, 80, 400, 100 }
```

Comment:
Has the extruder really been off by 10% up to now? Where did the 110 value come from?

Reply:
I honestly don't know. I noticed quite severe overextrusion, did an extruder calibration, was astonished by it eating 11cm on a 10cm command, tried 100 as value, now it prints better. not perfect, but...eh.